data/datasets.js

```diff
@@ -1,10 +1,17 @@
+import content from '../content/interactive.md';
+
 const INCOMPLETE_YEAR_NOTE = 'Data from the shaded year is incomplete.';
 const OIS_INCOMPLETE_YEARS = [2015];
 const CUST_DEATHS_INCOMPLETE_YEARS = [];
 
+const {
+  attributes: { datasets: cmsDatasets },
+} = content;
+const lastUpdatedFromSlug = slug => cmsDatasets.find(dataset => dataset.link === `datasets/${slug}`)?.date;
```

Comment:
I don't love how this introduces some coupling. In order for us to have `lastUpdated` values, we need to ensure that:

- the content from the CMS continues to live at `'../content/interactive.md'`
- the links to the datasets continue to be stored in a `link` attribute in the content Markdown file
- these dataset pages continue to live at `datasets/${slug}` URLs

It feels like this is an improvement over manually updating these strings every month though, and maybe these things are unlikely to change anytime soon. Perhaps this is a good incremental improvement until we introduce some automation around these dates. Let me know if you can think of better approaches!